lib/utils/domFns.js

```diff
@@ -109,13 +109,28 @@ export function offsetXYFromParent(evt: {clientX: number, clientY: number}, offs
   return {x, y};
 }
 
-export function createCSSTransform({x, y}: {x: number, y: number}): Object {
-  // Replace unitless items with px
-  return {[browserPrefixToKey('transform', browserPrefix)]: 'translate(' + x + 'px,' + y + 'px)'};
+export function createCSSTransform({x, y}: ControlPosition, positionOffset: PositionOffsetControlPosition): Object {
+  let translation;
+  if (positionOffset && (positionOffset.x !== 0 || positionOffset.y !== 0)) {
+    const defaultX = `${(typeof positionOffset.x === 'string') ? positionOffset.x : positionOffset.x + 'px'}`;
+    const defaultY = `${(typeof positionOffset.y === 'string') ? positionOffset.y : positionOffset.y + 'px'}`;
+    translation = `translate(${defaultX}, ${defaultY}) translate(${x}px,${y}px)`;
+  } else {
+    translation = `translate(${x}px,${y}px)`;
+  }
+  return {[browserPrefixToKey('transform', browserPrefix)]: translation };
 }
 
-export function createSVGTransform({x, y}: {x: number, y: number}): string {
-  return 'translate(' + x + ',' + y + ')';
+export function createSVGTransform({x, y}: ControlPosition, positionOffset: PositionOffsetControlPosition): string {
```

Comment:
Really feels like this could be merged w/ the previous function, perhaps with a param (default `'px'`) to determine translate unit suffix.

Reply:
Sounds good